pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -160,9 +129,12 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 	ownedPrs := list.Items
-	activeOrLastPRForChecks := c.activeOrLastPRForChecks(ctx, wl, relevantChecks, ownedPrs)
+	activeOrLastPRForChecks, latestStalePR := c.activeOrLastPRForChecks(ctx, wl, relevantChecks, ownedPrs)
```

Comment:
I would like to better understand if / why we need `latestStalePR`. We synced with @PBundyra who is yet looking into this.